core/src/main/groovy/noe/server/Tomcat.groovy

```diff
@@ -641,47 +641,47 @@ class Tomcat extends ServerAbstract implements WorkerServer {
     this.updateConfReplaceRegExp('server.xml', '<Connector port="8009" protocol="AJP/1.3"', '<Connector port="' + Integer.valueOf(8009 + offset) + '" protocol="AJP/1.3"', true, true)
   }
 
-  @Deprecated
```

Comment:
Not sure if we want to undeprecate this method, as it's using some updateConfReplaceRegExp() and a dummy comment to update the configuration.